cdap-common/src/main/java/io/cdap/cdap/common/internal/remote/RunnableTaskModule.java

```diff
@@ -18,20 +18,30 @@
 
 import com.google.inject.AbstractModule;
 import io.cdap.cdap.common.conf.CConfiguration;
+import org.apache.twill.discovery.DiscoveryService;
+import org.apache.twill.discovery.DiscoveryServiceClient;
 
 /**
  * Module for Runnable tasks.
  */
 public class RunnableTaskModule extends AbstractModule {
 
   private final CConfiguration cConf;
+  private final DiscoveryService discoveryService;
+  private final DiscoveryServiceClient discoveryServiceClient;
 
-  public RunnableTaskModule(CConfiguration cConf) {
+  public RunnableTaskModule(CConfiguration cConf,
+                            DiscoveryService discoveryService,
+                            DiscoveryServiceClient discoveryServiceClient) {
     this.cConf = cConf;
+    this.discoveryService = discoveryService;
+    this.discoveryServiceClient = discoveryServiceClient;
   }
 
   @Override
   protected void configure() {
     bind(CConfiguration.class).toInstance(cConf);
```

Comment:
The CConfiguration should be provided by the `ConfigModule` when creating the Injector, not from this module.

Reply:
Offline discussed, updated 👍